littlechef/runner.py

```diff
@@ -528,21 +532,26 @@ def _readconfig():
     except (ConfigParser.NoSectionError, ConfigParser.NoOptionError):
         env.follow_symlinks = False
 
+    env.cookbook_search_paths=[]
     try:
         env.berksfile = config.get('kitchen', 'berksfile')
     except (ConfigParser.NoSectionError, ConfigParser.NoOptionError) as e:
         env.berksfile = None
     else:
+        env.cookbook_search_paths=[]
         try:
             env.berksfile_cookbooks_directory = config.get('kitchen', 'berksfile_cookbooks_directory')
-            littlechef.cookbook_paths.append(env.berksfile_cookbooks_directory)
+            env.cookbook_search_paths.append(env.berksfile_cookbooks_directory)+'/berks_cookbooks'
         except (ConfigParser.NoSectionError, ConfigParser.NoOptionError) as e:
             if env.berksfile:
```

Comment:
I would put this if before trying to set env.berksfile_cookbooks_directory.
Also,  why the hardcoded `+'/berks_cookbooks'`?